### ISSUE TITLE
Updates table headers: the Update All column

### ISSUE THREAD

**Issue by afercia** [obenland/shiny-updates#118]
Similar to #114 table headers should describe the type of content of that column. This is not just for abstract semantics but because that information is used by software to communicate users the structure of the table. For example, see in the screenshot below. Navigating through table cells, each time screen readers read the content of the last cell in a row, they will read out also the related table header.
Hearing each time "Update All Update {thing to update name}" wouldn't be so great.

Worth reminding that tables with `<scope>` attributes establish logical relationships between column/row headers and the table cells. They shouldn't be used for visual purposes but exclusively to represent tabular data. So when deciding to use a table, then the table should be used properly.

<img width="1440" alt="screen shot 2016-05-19 at 09 39 30" src="https://cloud.githubusercontent.com/assets/1682452/15386301/4e51499c-1da7-11e6-81b4-b0310e33dab3.png">

**Comment by swissspidy** [comment]
Off the top of my head I see a few options:

* Remove `scope` attribute from table heading
* Test changing `scope` to `rowgroup`
* Move `Update All` to outside the list table where bulk actions usually reside  
 Table headings could then be: `Update` | `Type` | `Action`

@afercia Any suggestions?

**Comment by afercia** [comment]
I guess the easier one would be 3 :) 
Ideally, in 2016 tables should be used just for grids of data. Using them for layout purposes would be just repeating the same mistakes already done in the past but I understand this would require a major refactoring. Thinking at "what" this block of content represents, well this is a _list_ of updates and should probably be marked up as a list (`<ul>` or `<dl>` maybe). Not something that can be done now I guess, and probably would need a major re-thinking so fine with a table for now :)

**Comment by swissspidy** [comment]
Absolutely agree with you regarding the purpose of tables. Since WordPress already uses list tables for available updates and elsewhere, I didn't want to reinvent the wheel.

I'll create a PR with method 3.

**Comment by swissspidy** [comment]
Alright, I have something working. Looks like this:

<img width="1262" alt="screen shot 2016-05-19 at 12 36 31" src="https://cloud.githubusercontent.com/assets/841956/15390947/f0c13374-1dbe-11e6-9cd4-81740bf61f25.png">

See #119, which also fixes #111.
